buildSrc/src/integTest/groovy/org/elasticsearch/gradle/JdkDownloadPluginFuncTest.groovy

```diff
@@ -132,7 +132,7 @@ class JdkDownloadPluginFuncTest extends AbstractGradleFuncTest {
         }
 
         then:
-        result.tasks.size() == 10
+        result.tasks.size() == 3
         result.output.count("Unpacking linux-12.0.2-x64.tar.gz using SymbolicLinkPreservingUntarTransform.") == 1
```

Comment:
Should we reference the class `simpleName` here like we've done elsewhere to make refactoring less painful?

Reply:
actually just got bitten by that during my refactoring :P . Fixed